crates/miden-lib/asm/miden/account.masm

```diff
@@ -705,3 +705,59 @@ export.was_procedure_called
     swapdw dropw dropw swapw dropw movdn.3 drop drop drop
     # => [was_called]
 end
+
+#! Returns the number of procedures in the current account.
+#!
+#! Inputs:  []
+#! Outputs: [num_procedures]
+#!
+#! Where:
+#! - num_procedures is the number of procedures in the current account.
+#!
+#! Invocation: exec
+export.get_num_procedures
```

Comment:
We should add this and `get_procedure_root` procedures to the table of kernel procedures (i.e., [here](https://github.com/0xMiden/miden-base/blob/next/docs/src/protocol_library.md)).